src/pages/home/home.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+// import { videos } from '../../backend/db/products';
+import { Navbar } from '../../components/allComponents';
+import "./home.css";
+import { products } from '../../backend/db/products';
+import "../../css/videoCard.css"
+
+
+function Home() {
+  return (
+    <>
+    <Navbar />
+
+    <section className='main-page'>
+
+      {/* sidebar */}
+      <div className="col1">
+     <ul>
+       <li><i className="fas fa-home"></i><span>Home</span> </li>
+       <li><i className="fas fa-folder"></i><span>Playlists</span> </li>
+       <li><i className="fas fa-heart"></i><span>Liked </span></li>
+       <li><i class="fas fa-clock"></i><span>Watch Later</span> </li>
+       <li><i class="fas fa-history"></i><span>History</span> </li>
+     </ul>
+      </div>
+
+      {/* videos */}
+      <div className="col2">
+        <div className='btn-category'>
+          <button>All</button>
+          <button>Ice Cream</button>
+          <button>Cake</button>
+          <button>Choclate</button>
+        </div>
+        <div className='videosListing'>
+
+        {
+          products.map(item=>{
```

Comment:
I think we're not supposed to get products data like this. we should fetch data using API. On Mockbee docs, we have routes defined for each data. so we are supposed to fetch the data & store it in context ( state ) & then use it in our components

Reply:
I have updated it ,thankyou for letting me